apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/ListAssertionSamples.kt

```diff
@@ -0,0 +1,57 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class ListAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
+    private val isGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+
+    @Test
+    fun getFeature() {
+        val list = listOf(1, 2, 3)
+
+        expect(list) get 0 isLessThan 2
```

Comment:
Please include the comments as they exist in ch.tutteli.atrium.api.fluent.en_GB.samples.ListAssertionSample because they provide valuable information to better understand how Atrium works. I suggest you try the following style but maybe you have a better idea?
```kotlin
expect(list) get 0 isLessThan 2 isGreaterThan 0
//                |            | subject is still of type Int (still 1)
//                | subject is now of type Int (actually 1)          
                
```

Reply:
Sure, will do. Will update the pull request.